.changeset/sour-spiders-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-bitbucket-server': patch
+---
+
+feat(catalog): add optional branch to bitbucket-server
```

Comment:
```suggestion
Added optional branch to the Bitbucket Server entity provider filters
```

As these get rolled up into the changelog it's nice to have them be people friendly. 👍